src/matrix/sparse-matrix.h

```diff
@@ -338,6 +342,15 @@ void FilterSparseMatrixRows(const SparseMatrix<Real> &in,
                             const std::vector<bool> &keep_rows,
                             SparseMatrix<Real> *out);
 
+/// Outputs a SparseMatrix<Real> containing all the rows within the range
```

Comment:
<pre><code>
/// Outputs a SparseMatrix<Real> containing all the rows of matrix "in" 
/// in the specified range [row_begin, row_end).
/// It is significantly faster alternative for filtering contiguous rows
/// compared to FilterSparseMatrixRows().
</code></pre>